evaluator/src/main/kotlin/SourceTreeResolver.kt

```diff
@@ -46,5 +46,7 @@ fun interface SourceTreeResolver {
 
             downloadDir
         }
+
+        fun forLocalDir(dir: File) = SourceTreeResolver { dir }
```

Comment:
Nit: As you use "Repository" instead of "Repo" in the function name above, maybe also spell out "Directory" as part of the function name here?